hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java

```diff
@@ -354,4 +354,16 @@ Object authenticationCustom(Data credentials, @Nullable String uuid, @Nullable S
     @Request(id = 18, retryable = false, response = ResponseMessageConst.VOID, event = {EventMessageConst.EVENT_PARTITIONS})
     @Since(value = "1.5")
     void addPartitionListener();
+
+    /**
+     * @param target  target address to create proxies on
```

Comment:
is this needed for dummy client?

Reply:
Yes, it is needed. It is mainly introduced for replicated map/lite member problem. 
The client is trying to send to proxies to a non-lite member even though it's owner is a lite member.